deepmd/tf/fit/fitting.py

```diff
@@ -231,7 +235,7 @@ def serialize_network(
                 # initialize the network if it is not initialized
                 fittings[network_idx] = FittingNet(
                     in_dim=in_dim,
-                    out_dim=1,
+                    out_dim=out_dim,
```

Comment:
This is to ensure the serialize/deserialize is consistent when using `serialize_network`